docs/ui/storage/fragments/react-native/s3-album.md

```diff
@@ -1,27 +1,16 @@
-`S3Album` renders a list of `S3Image` and `S3Text` objects:
+`S3Album` renders a list of `S3Image` objects:
 
-![Image](~/images/S3Album_and_code.png)
-
-```jsx
-import { S3Album } from 'aws-amplify-react-native';
-
-render() {
-    return <S3Album path={path} />
-```
+![S3Album Example](~/images/s3album-react-native.png)
```

Comment:
Lets remove this image.Lets create a code block with the sample code in the image. Also, add an image on top of this code block with how S3Album look on ios and android simulators.